lib/install/phases/pull.go

```diff
@@ -150,7 +150,7 @@ func (p *pullExecutor) pullUserApplication() error {
 		DstApp:      p.LocalApps,
 		Package:     *p.Phase.Data.Package,
 	})
-	if err != nil {
+	if err != nil && !trace.IsAlreadyExists(err) {
```

Comment:
these need to be re-entrant.

Reply:
Sometimes you point out really great stuff by commenting on your own PR, however, I feel this is a comment that is better suited in the source. It answers the why of a particular line of code and isn't a simple restatement of the code.

Why is AlreadyExists an acceptable error here? 
Answer: Needs to be re-entrant.